lib/apiservers/service/restapi/handlers/vch_create.go

```diff
@@ -38,10 +41,16 @@ import (
 	"github.com/vmware/vic/lib/install/data"
 	"github.com/vmware/vic/lib/install/management"
 	"github.com/vmware/vic/lib/install/validate"
+	"github.com/vmware/vic/lib/install/vchlog"
 	"github.com/vmware/vic/pkg/ip"
+	viclog "github.com/vmware/vic/pkg/log"
 	"github.com/vmware/vic/pkg/version"
 )
 
+const (
+	LogFile = "vic-machine.log" // name of local log file
```

Comment:
Is `LogFile` used in another pkg? If not, it shouldn't be exported and should be declared as `logFile`.